app/routes/meeting-api.js

```diff
@@ -1,11 +1,44 @@
 var express = require('express');
+var jwt = require('jsonwebtoken');
 var router = express.Router();
 
 var meetingController = require('../controllers/meetingController');
 
 router.get('/', function (req, res) {
     res.render('meeting');
 });
+
+router.use(function (req, res, next) {
+    // check header or url parameters or post parameters for token
+    var token = req.body.token || req.query.token || req.headers['x-access-token'];
+    // decode token
+    if (token) {
+        // verifies secret and checks exp
+        jwt.verify(token, 'xingyunzh-secret', function (err, decoded) {
+            if (err) {
+                return res.json({success: false, message: 'Failed to authenticate token.'});
+            } else {
+                // if everything is good, save to request for use in other routes
+                req.decoded = decoded;
+                next();
+            }
+        });
+        console.log("Token OK");
+    } else {
+        // if there is no token
+        // return an error
+        console.log("No token " + token);
+        res.render('login');
+    }
+});
```

Comment:
You are right, so i make the token parser a module and have already put it into /util/shared